test/cli/not_stripe_packages/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# We currently populate core::File::Flags::isPackage only via the filename,
+# regardless of whether the `--stripe-packages` flag has been passed.
+#
+# This makes it easy to have behavior that leaks `--stripe-packages`-specific
+# behavior into the pipeline. Long term, we should figure out how to fix this.
```

Comment:
Great point here, do we already have an issue tracking this?

Reply:
#8622